src/lib/workspace/DownloadTab/index.tsx

```diff
@@ -140,3 +200,38 @@ export default function DownloadTab({
     </div>
   );
 }
+
+function getDataAccessDeclaration(
+  studyAccess: string,
+  isGuest: boolean,
+  hasPermission: boolean = false
+): string {
+  const DATA_ACCESS_STUB = `Data downloads for this study are ${studyAccess.toLowerCase()}. `;
+  const PUBLIC_ACCESS_STUB = 'You can download the data without logging in.';
+  const LOGIN_REQUEST_STUB =
+    'You must register or log in and request access to download data;';
+  const CONTROLLED_ACCESS_STUB =
+    ' data can be downloaded immediately following request submission. ';
+  const PROTECTED_ACCESS_STUB =
+    ' data can be downloaded after the study team reviews and grants you access. ';
+  // const REQUEST_ACCESS_STUB = 'Click here to request access.';
+  const ACCESS_GRANTED_STUB =
+    'You have been granted access to download the data.';
+  // const ACCESS_PENDING_STUB = 'Your data access request is pending.';
+
+  let dataAccessDeclaration = DATA_ACCESS_STUB;
+  if (studyAccess === 'Public') {
+    return (dataAccessDeclaration += PUBLIC_ACCESS_STUB);
+  } else if (isGuest || (!isGuest && !hasPermission)) {
+    dataAccessDeclaration += LOGIN_REQUEST_STUB;
+    return (dataAccessDeclaration +=
+      studyAccess === 'Controlled'
```

Comment:
Idea for improved resilience: instead of checking whether `studyAccess === 'Controlled'`, you could use the [getStudyRequestNeedsApproval](https://github.com/VEuPathDB/web-study-data-access/blob/1f04456696ef990eddc3716fd6ba5fcdbcb4cdb9/src/shared/studies.ts#L54) utility, which returns the value `'0'` if request approval isn't required.
```suggestion
      getStudyRequestNeedsApproval(studyRecord) === '0'
```